packages/calcite-components/src/components/slider/slider.scss

```diff
@@ -221,29 +239,30 @@
 .thumb:focus,
 .thumb--active {
   .handle {
-    @apply bg-brand;
+    background-color: var(--calcite-color-brand);
     box-shadow: 0 0 8px 0 rgb(0 0 0 / 16%); // shadow 1 press
   }
 }
 
 .thumb:hover.thumb--precise::after,
 .thumb:focus.thumb--precise::after,
 .thumb--active.thumb--precise::after {
-  @apply bg-brand;
+  background-color: var(--calcite-color-brand);
 }
 
 .track {
   @apply relative h-0.5 rounded-none;
-  background-color: var(--calcite-color-border-2);
+  background-color: var(--calcite-slider-track-color, var(--calcite-color-border-2));
   transition: all var(--calcite-internal-animation-timing-medium) ease-in;
 }
 
 .track__range {
-  @apply bg-brand
-  absolute
+  @apply absolute
   top-0
   h-0.5;
+  background-color: var(--calcite-slider-track-fill-color, var(--calcite-color-brand));
```

Comment:
this might also actually need to be changed to be a shared token with the lines above.